network-api/networkapi/mozfest/templates/fragments/blocks/mozfest_cta_block.html

```diff
@@ -0,0 +1,17 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+    <div class="tw-py-16 tw-dark{% if self.dark_background %} tw-bg-black{% endif %}">
```

Comment:
Oh. So the `tw-dark` class actually sets the font-color? Interesting. Based on the [Tailwind docs](https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually) I was expecting this to be only a "flag" class. 

I guess it might be a little more explicit to directly set the font-color on the inner box is we don't really mean this to be changed based on dark/light mode, but I guess we don't need an extra PR for it now.